package.json

```
@@ -6,7 +6,7 @@
     "private": true,
     "dependencies": {
         "@babel/runtime": "^7.21.0",
-        "@lingui/react": "^3.17.2",
+        "@lingui/react": "^v4.0.0-next.5",
```

The official v4 release is here, so you can already do the following :)

```suggestion
        "@lingui/react": "^v4.0.0",
```

https://github.com/lingui/js-lingui/releases/tag/v4.0.0